management-microservice/src/test/java/com/nearsoft/fgaribay/mgmt/microservice/MicroserviceTest.java

```diff
@@ -0,0 +1,87 @@
+package com.nearsoft.fgaribay.mgmt.microservice;
+
+import com.nearsoft.fgaribay.mgmt.ProductRepository;
+import com.nearsoft.fgaribay.mgmt.exceptions.ProductDataException;
+import com.nearsoft.fgaribay.mgmt.model.Product;
+import com.nearsoft.fgaribay.mgmt.model.ServiceRequest;
+import com.nearsoft.fgaribay.mgmt.model.ServiceResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ContextConfiguration
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class MicroserviceTest {
+
+  @MockBean
+  @Qualifier("productRepository")
+  ProductRepository productRepository;
+
+  @Autowired ProductManagementService service;
+
+  @Before
+  public void setup() {}
```

Comment:
Empty setup methods are not needed or required. Try removing them from your tests and they should work in the same way.